octoprint_mrbeam/templates/wizard/wizard_whatsnew_0.jinja2

```diff
@@ -1,8 +1,19 @@
-<h3>{{ _("Improved Camera") }}</h3>
+<h3 style="text-transform: uppercase; color: #e25303">
+    <img id="ds_title_stars" src="plugin/mrbeam/static/img/beta_dialog/stars.svg" alt="">
+    <span>{{ _("GCode auto-deletion") }}</span>
+</h3>
 
 <div class="wizard-content">
-    <p><strong>{{ _('The camera is much more reliable now!') }}</strong></p>
-    <p>{{ _('You will feel an improvement specially when using dark materials or with difficult light conditions, thanks to the self-adjusting shutter speed. ') }}</p>
-    <p>{{ _('Check the status of your camera in the new camera settings panel:') }}</p>
-    <img class="wizard-big-img wizard-shadowed-img" src="/plugin/mrbeam/static/img/whatsnew_dialog/camera_status.png">
+    <form class="form-horizontal">
+        <div class="control-group">
+            <div>
+                <p>{{ _('Every time you run a laser job, a GCode file is automatically created. We recommend to enable the auto-deletion of GCode files so that your Mr Beam doesn\'t get too loaded and it can work faster. With this enabled, only the last 25 GCode files will be stored.') }} </p>
+                <p>{{ _('You can alternativelly delete the GCode files manually in the Design Library.') }} </p>
+                <label class="checkbox">
+                    <input type="checkbox" data-bind="checked: gcodeAutoDeletion">
```

Comment:
I'm surprised that this works. @khaledsherkawi knowing you, I'm sure you tested it. Still, can you please confirm, that it works?

Reply:
It's the same concept you used for the beta whats new where I changed the IDs of both elements (beta whats new and stable whats new) and bind them to the WizardGcodeDeletionViewModel